src/patterns/kinesis-lambda.ts

```diff
@@ -40,8 +41,7 @@ import { toAwsErrorHandlingProps } from "../utils/lambda";
  * existingKinesisStream: { externalKinesisStreamName: "KinesisStreamFromAnotherStack" }
  * ```
  */
-export interface ExistingKinesisStream {
-  logicalIdFromCloudFormation?: string;
+export interface ExistingKinesisStream extends GuMigratingResource {
```

Comment:
Some of the docs (in the comment above this interface) need to be updated due to this change